.github/workflows/ci.yml

```
@@ -16,17 +20,51 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
+      - name: Cache APT packages
+        uses: actions/cache@v3
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)